### DISCUSSION TITLE
Selective records download [Oct 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4036] [Discord Threads]
> 🙋‍♂️ **prasanth**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### For selective download of records, you can apply filter & download. Only filtered records will be saved in CSV / XLSX.    
####    
#### Deleting filtered data, currently is per page. There is no easy way to delete all filtered records in one go.  
#  
> 🙋‍♂️ **prasanth**  
#### ohh ok <@857998597474484295>    
####    
#### there a typo in my first line of msg actually   
#### i have mentioned it as download instead of delete   
####    
#### anyway i understood that it is not currently avaiable   
####    
#### thanks for the reply  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1024540766212726806)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
